spec/models/norm_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Norm, type: :model do
+  describe '#name' do
+    subject(:name) { described_class.create(name: 'full_time').name }
+
+    it { is_expected.to eq 'full_time' }
+  end
```

Comment:
You're basically verifying whether the model has a `name` attribute, I don't think that brings much value.

Reply:
Not so much value, but also not so much cost of doing :)